src/pages/StartupsPage.tsx

```
@@ -0,0 +1,266 @@
+          </svg>
+        </div>
+
+        <div className="container mx-auto px-4 sm:px-6 lg:px-8">
+          <div className="max-w-3xl mx-auto text-center">
+            <motion.h2 
+              initial={{ opacity: 0, y: 20 }}
+              whileInView={{ opacity: 1, y: 0 }}
+              viewport={{ once: true }}
+              className="text-3xl md:text-4xl font-extrabold mb-6 text-white"
+            >
+              Saisissez votre chance et candidatez dès maintenant !             </motion.h2>
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)